### ISSUE TITLE
Align search dropdown look & behaviour

### ISSUE THREAD

**Issue by cuserox** [tokens-studio/figma-plugin#2786]

**Comment by six7** [comment]
I agree, we should move this to the DS repository - we're making use of something similar in Studio as well, that's a great contender for a shared utility that we re-use across implementations.
